great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -399,7 +399,7 @@ def get_domain_records(
                     subset=[column_A_name, column_B_name],
                 )
             else:
-                if ignore_row_if != "never":
+                if ignore_row_if != "neither":
                     raise ValueError(
```

Comment:
@alexsherstinsky We need to differentiate between `column_pair` and `multicolumn` otherwise this will break expectations where `ignore_row_if="never"` is used in the documentation.

Reply:
@NathanFarmer Doesn't this PR do exactly that?